maps/torch/items/summarydocuments.dm

```diff
@@ -1,130 +1,114 @@
 	name = "anti-fraternization memo"
-	desc = "An \"internal use only\" memo sent from the Senior Enlisted Advisor to infirmary staff, informing them that, yes, inappropriate relationships are forbidden for corpsmen also, and that continued rule violations may result in NJP's, along with a reminder to focus on the provision of medical care."
+	desc = "An \"internal use only\" memo sent from the Senior Enlisted Advisor to all infirmary staff informing them that, yes, inappropriate relationships are also forbidden for corpsmen and medical technicians. It highlights that further instances of fraternization can and will result in non-judical punishment and reminds all staff that the provision of medical care is their first priority."
 
 /obj/item/documents/generic/sol/bunnybirdissues
 	name = "bridge officer hallucination issues"
-	desc = "An \"internal use only\" memo sent from the Chief Medical Officer to the Executive Officer detailing the need for downtime and recovery for bridge staff after numerous reports of certain bridge officers seeing giant space birds after too long at the helm, and a reminder for them to be sent for counselling."
+	desc = "An \"internal use only\" memo sent from the Chief Medical Officer to the Executive Officer detailing the need for downtime and recovery for bridge staff after numerous reports of certain bridge officers (whose names are redacted from the report) seeing \"giant space birds\" after too long at the helm, and reminding  them to be sent for psychological counseling at their earliest convenience."
 
 /obj/item/documents/generic/sol/bunnytraining
 	name = "bridge officer qualification status"
-	desc = "An \"internal use only\" memo sent from the Commanding Officer to heads of department reminding them that bridge officers are qualified in communications and bridge operations and do not, infact, double as naval infantry and should not be deployed as such during minor security incidents after numerous reports of broken legs and feet being shot."
+	desc = "An \"internal use only\" memo sent from the Commanding Officer to the heads of staff reminding them that bridge officers are qualified in communications and bridge operations only and do not, in fact, double as naval infantry. They should not be deployed like such during minor security incidents, citing numerous reports of broken legs and feet being shot due to mishandling of weapons."
 
 /obj/item/documents/generic/sol/comm
 	name = "ZA-Common reminder"
-	desc = "A memo that has seen better days. It's been crumpled up, dried in coffee and put back up by some vandals. From what's left, it reads: 'Speaking good ZA-Common is a must to be an excellent head of staff. Please exercise moderation with accents.'"
-
-/obj/item/document/generic/sol/holocallloiter
-	name = "holocall etiquette"
-	desc = "Some \"internal use only\" reports, requesting people that if a holocall lingers, they should make a point to end it."
```

Comment:
Removing this one as well for similar reasons. That, and it's a bit redundant; I've never seen a holocall drag on unnecessarily long myself.

Reply:
Agreed as above.